nncf/common/graph/operator_metatypes.py

```diff
@@ -100,15 +101,15 @@ def wrap(obj: Type[OperatorMetatype]) -> Type[OperatorMetatype]:
             if cls_name is None:
                 cls_name = obj.__name__
             super_register(obj, cls_name)
-            op_names = obj.get_all_aliases()
-            for name in op_names:
-                if name in self._op_name_to_op_meta_dict and not obj.subtype_check(self._op_name_to_op_meta_dict[name]):
-                    raise nncf.InternalError(
-                        "Inconsistent operator metatype registry - single patched "
-                        "op name maps to multiple metatypes!"
-                    )
-
-                self._op_name_to_op_meta_dict[name] = obj
+            if not is_subtype:
```

Comment:
```suggestion
            if is_subtype:
                return obj
            ....
```

Reply:
I'm not sure that code like this

```python
if is_subtype:
    return obj
...
return obj
```

is better that 

```python
if not is_subtype:
   ...
return obj
```

Reply:
It is believed that less indentation increase code readability, for example to understand how this code is working for `is_type == True` one should skip if and check nothing is called after it. But do as you wish